### PR TITLE
feat: leaderboard markup structure

### PR DESCRIPTION
**Pull request by Amrhub**
# Changes made
* add leaderboard markup
* add bit styling for the desired layout according to this wireframe: 
image

## Project requirements
[Project requirements](https://github.com/microverseinc/curriculum-javascript/blob/main/leaderboard/m1_basic_structure.md)

### CODE REVIEW

File: dist/index.html [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+            <section class=".col col--add-score">
+                <h2>Add your score</h2>
+                <form class="col--add-score__form">
+                    <label for="name" hidden>Enter your name</label>
+                    <input type="text" name="name" id="name" placeholder="Your name">
+                    <label for="score" hidden>Enter your score</label>
+                    <input type="number" name="number" id="score" placeholder="Your score" min="0">
+                    <button type="submit" class="btn-submit">Submit</button>
+                </form>
+            </section>
```
Comment (henatan99) [inline dist/index.html]: - [x] One of the objectives of this milestone is to practice JS ES6 modules, functions, syntaxes, and DOM. I think the plain HTML phrase is used to emphasize that styling using CSS is mandatory for this milestone. Kindly construct and render components in JS DOM which are children of the root HTML element. :pray:

**Review by henatan99 — CHANGES_REQUESTED** [review]
Hi @Amrhub , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!
<FEEL FREE TO CHANGE THIS MESSAGE TO YOUR PERSONAL WORDS>


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
- [x] OPTIONAL: Kindly try to commit more often as it helps track the changes you made more easily. :pray: 
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: dist/index.html (hunk 1 above)
Comment (Amrhub) [reply dist/index.html]: I added the list dynamically is that what you meant 😓  ?

Comment (henatan99) [reply dist/index.html]: This issue has been addressed well. Nice job! :clap: :clap: :clap:

**Review by henatan99 — APPROVED** [review]
Hi @Amrhub , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**